### ISSUE TITLE
[Quest] [Val'sharah] Cenarius, Keeper of the Grove -Impossible to start chain quest-

### ISSUE THREAD

**Issue by talosbane00** [Project-AuroraWoW/Project-Aurora-Bug-Tracker#549]
[https://www.wowhead.com/quest=38305/cenarius-keeper-of-the-grove](url)

Nothing happen if you try to deliver quest to Malfurion.

Actually it is not possible to start Val'sharah chain quests.

**Comment by Aerosnail** [comment]
I managed to complete the quest just now. Malfurion's pathing is a bit wonky but once he gets to where Cenarius is lying down I could complete it normally, and pick up the follow-up quest. Maybe there's some hidden requirements that don't let you complete it?

**Comment by talosbane00** [comment]
Agreed. Malfurion have huge pathfinding issues. But that is not the real issue. After He turn back in "Human" form a Yellow question mark appears on him but if I try to complete quest nothing happen. Even if I abandon quest and take it again.

P.s. after every issue I usually delete cache folder and/or abandon and take again quest.
